src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -492,7 +498,23 @@ public static String injectContextDependentValuesForTest(String content) {
                                StringHelper.truncateLongId(TestProperties.TEST_ADMIN_ACCOUNT))
                       .replace("<!-- nexthour.date -->", TimeHelper.formatDate(TimeHelper.getNextHour()))
                       .replace("<!-- now.datetime -->", TimeHelper.formatTime12H(now))
-                      .replace("<!-- now.datetime.comments -->", TimeHelper.formatDateTimeForComments(now));
+                      .replace("<!-- now.datetime.comments -->", TimeHelper.formatDateTimeForComments(now, 0));
+    }
+
+    private static TimeZone getTimeZone(String content) {
+        //Searches for last String of pattern "UTC+0800" in the content.
+        Pattern pattern = Pattern.compile(REGEX_TIMEZONE_OFFSET);
+        Matcher matcher = pattern.matcher(content);
+        //set default time zone offset.
+        String timeZoneOffset = "+0000";
+        while (matcher.find()) {
+            timeZoneOffset = matcher.group(1);
+        }
+        return TimeZone.getTimeZone("GMT" + removeUtc(timeZoneOffset));
+    }
+
+    private static String removeUtc(String timeZoneOffset) {
+        return timeZoneOffset.substring(0, 2);
```

Comment:
No, I am searching for the last string with a pattern as `UTC+xxxx` and then extracting `+xxxx` from it.For example, `timeZoneOffset` is `UTC-0330` then this will return `-0330`.